go.mod

```diff
@@ -19,8 +19,8 @@ require (
 	github.com/coreos/yaml v0.0.0-20141224210557-6b16a5714269 // indirect
 	github.com/digitalocean/godo v1.45.0
 	github.com/flatcar-linux/container-linux-config-transpiler v0.9.3-0.20220208152502-6e8303479682
-	github.com/flatcar-linux/ignition v0.36.1
-	github.com/flatcar-linux/ignition/v2 v2.2.1-0.20220107090316-32908ec8bade
+	github.com/flatcar-linux/ignition v0.36.2-0.20220107144332-c956607ca6e9
+	github.com/flatcar-linux/ignition/v2 v2.2.1-0.20220302151613-dcd27a7791fb
```

Comment:
This is a reference to the temporary branch, or?

Reply:
yes, this is the current reference for https://github.com/flatcar-linux/ignition/pull/23 - should be updated once merged and tagged.

Reply:
The last commit on our `main` is now `cb95c51122f5a95cea4ed042677933c588fded18`